editor/connections_dialog.cpp

```diff
@@ -1130,7 +1138,7 @@ void ConnectionsDock::_signal_menu_about_to_popup() {
 		}
 	}
 
-	signal_menu->set_item_disabled(slot_menu->get_item_index(DISCONNECT_ALL), disable_disconnect_all);
+	signal_menu->set_item_disabled(signal_menu->get_item_index(DISCONNECT_ALL), disable_disconnect_all);
```

Comment:
I haven't tested, but this is obviously a bug.

Reply:
Might be the cause of "Disconnect" sometimes only working from the button on the bottom. Hooray!